docs/source/bag-api.rst

```diff
@@ -62,9 +62,20 @@ Top-level functions
    map_partitions
    zip
 
+Random Sampling
+---------------
```

Comment:
I would get rid of this title, and just add `random.choices` and `random.sample` to the list above since these functions fit well with the "top-level functions" and the left-hand-side navigation has duplicated titles:

![image](https://user-images.githubusercontent.com/4806877/83415063-83f98a80-a3ec-11ea-9cb3-c942e04cf40c.png)

Reply:
Here we go. thanks for the suggestion